README.md

```diff
@@ -2,3 +2,65 @@
 
 URL shortening website made as part of a paired code challenge.
 
+## Installation & Usage
+
+### Prerequisites
+
+* You must have Python 3.9.1 installed.
+
+### Installation
+
+1. Clone the repo using `git clone git@github.com:rxdvd/shortlify.git`
+2. Enter the repo directory `cd shortlify`
+3. Create the virtual environment `pipenv --python 3.9.1`
+4. Enter the virtual environment `pipenv shell`
+5. Install dependencies `pipenv install --dev`
```

Comment:
very nice and clear instructions - although I had to `pipenv install colorama` and and the .env